jbt-it/neomdb
# 455 server add application portal

How to test:
- create a new application with /bewerbungen
- check the db if the trainee application has been saved
- run the applicaion test
